src/main/java/com/icthh/xm/ms/entity/repository/XmEntityRepository.java

```diff
@@ -30,130 +30,162 @@
     private final SpringXmEntityRepository springXmEntityRepository;
     private final TenantConfigService tenantConfigService;
 
+    @Override
     public XmEntity findOneByIdForUpdate(@Param("id") Long id) {
         return springXmEntityRepository.findOneByIdForUpdate(id);
     }
 
+    @Override
     public XmEntity findOneById(Long id) {
         return springXmEntityRepository.findOneById(id);
     }
 
+    @Override
     public Page<XmEntity> findAllByTypeKeyIn(Pageable pageable, Set<String> typeKeys) {
         return springXmEntityRepository.findAllByTypeKeyIn(pageable, typeKeys);
     }
 
+    @Override
     public XmEntityIdKeyTypeKey findOneIdKeyTypeKeyById(Long id) {
         return springXmEntityRepository.findOneIdKeyTypeKeyById(id);
     }
 
+    @Override
     public XmEntityIdKeyTypeKey findOneIdKeyTypeKeyByKey(String key) {
         return springXmEntityRepository.findOneIdKeyTypeKeyByKey(key);
     }
 
+    @Override
     public XmEntityStateProjection findStateProjectionById(Long id) {
         return springXmEntityRepository.findStateProjectionById(id);
     }
 
+    @Override
     public XmEntityStateProjection findStateProjectionByKey(String key) {
         return springXmEntityRepository.findStateProjectionByKey(key);
     }
 
+    @Override
     public Optional<XmEntityVersion> findVersionById(Long id) {
         return springXmEntityRepository.findVersionById(id);
     }
 
+    @Override
     public boolean existsByTypeKeyAndNameIgnoreCase(String typeKey, String name) {
         return springXmEntityRepository.existsByTypeKeyAndNameIgnoreCase(typeKey, name);
     }
 
+    @Override
     public List<XmEntity> findAll() {
         return springXmEntityRepository.findAll();
     }
 
+    @Override
     public List<XmEntity> findAll(Sort sort) {
         return springXmEntityRepository.findAll(sort);
     }
 
-    public List<XmEntity> findAll(Iterable<Long> longs) {
-        return springXmEntityRepository.findAll(longs);
+    @Override
+    public List<XmEntity> findAllById(Iterable<Long> longs) {
```

Comment:
ALARM! It's can broke leps!
Please rollback rename method.

Reply:
fixed